performancebench/src/main/scala/cognite/spark/performancebench/Main.scala

```diff
@@ -1,12 +1,16 @@
 package cognite.spark.performancebench
 
 import io.prometheus.client.exporter.HTTPServer
+import io.prometheus.client.hotspot.{DefaultExports => PrometheusDefaultExports}
 import org.log4s._
 
 object Main extends App {
   val logger = getLogger
   val metricsServer = new HTTPServer(8123, true)
+  PrometheusDefaultExports.initialize()
   try {
+    // We need to warm-up our Spark cluster to get fair metrics
+    Warmup.run()
```

Comment:
Isn't it a bit overkill to make this an object instead of just a local method?

Reply:
Probably 🤷‍♂